src/components/Button/Button.jsx

```diff
@@ -0,0 +1,64 @@
+import 'bulma/css/bulma.css';
+import cn from 'classnames';
+
+const ACTION_FIELD_ALPHABET = 'Sort alphabetically';
+const ACTION_FIELD_LENGTH = 'Sort by length';
+const ACTION_FIELD_REVERSE = 'Reverse';
+const ACTION_FIELD_RESET = 'Reset';
+
+function getGoodsToRend(goods, field) {
+  const goodsToRend = [...goods];
+
+  if (field) {
+    goodsToRend.sort((good1, good2) => {
+      switch (field) {
+        case ACTION_FIELD_ALPHABET:
+
+          return good1.name.localeCompare(good2.name);
+        case ACTION_FIELD_LENGTH:
+
+          return good1.name.length - good2.name.length;
+        default:
+          return 0;
+      }
+    });
+  }
```

Comment:
This func could be written in separate file as helper and it shoul take array of goods, field, isReversed (which should be defined as state as I mentioned previously).